lib/r10k/module/forge.rb

```diff
@@ -71,7 +73,9 @@ def sync(opts={})
         reinstall
       end
       maybe_delete_spec_dir
+      updated = true
```

Comment:
No, what I was thinking was something like:
```ruby
def sync(opts = {})
  updated = false
  if should_sync?
    updated = true
    case status
    when ...
    ....
    else
      updated = false
    end
    maybe_delete_spec_dir
  end
  updated
end
```

Maybe that's more confusing? I think as it's written now if the module should be synced, but `status` returns `:insync` then it will still be marked as updated??

Reply:
yeah, I know that's what you meant. I thought what I did would be clearer, but didn't realize that there was an `:insync` status to worry about. But yeah, not sure if your thing is more or less confusing than the original.

Reply:
okay, I think I like the explicitness of how it was originally, even though it's a little more repetitious.

Reply:
I put the original back.